### PR TITLE
Update styles for new case contact form

### LINKED ISSUE #5498: Minor UI updates to Case Contact Form
**Description**
Some minor UI changes to increase accessibility

1. Update all checkbox and radio button outlines in the unchecked state to be #757575 instead of #E5E5E5 
2. Right-align form buttons and make them title case
    1. On pages that include the “Save and Continue” button, this button should be the rightmost, and the “Back” button should be 10 pixels to the left
    2. On pages that include the "Submit" button, this button should be the rightmost, and the "Back" button should be 10 pixels to the left
    3. For the "New court report topic" in the admin portal, "Submit" and "Back" should be right aligned with "Submit" being the rightmost.
3. For Mobile view only:
    1. Reduce the size of the H1 “Record new case contact” to size H2
    2. Reduce the size of the H2 “Contact Details” to size H3

For reference, please see the [Figma](https://www.figma.com/file/QHOE6wHBESpsbxQmlnAZ2L/Case-Contact-Reporting-Process?type=design&node-id=148%3A16461&mode=design&t=iv2Vrg72rct7UwRQ-1) document.

### Questions? Join Slack!

We highly recommend that you join us in [slack](https://join.slack.com/t/rubyforgood/shared_invite/zt-21pyz2ab8-H6JgQfGGI0Ab6MfNOZRIQA) #casa channel to ask questions quickly and hear about office hours (currently Tuesday 5-7pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by Natblow**
### What github issue is this PR for, if any?
Resolves #5498 

### What changed, and why?

- [x]  Update all checkbox and radio button outlines in the unchecked state to be #757575 instead of #E5E5E5
- [x] Right-align form buttons and make them title case
    - [x]    On pages that include the “Save and Continue” button, this button should be the rightmost, and the “Back” button should be 10 pixels to the left
    - [x]   On pages that include the "Submit" button, this button should be the rightmost, and the "Back" button should be 10 pixels to the left
    - [ ]   For the "New court report topic" in the admin portal, "Submit" and "Back" should be right aligned with "Submit" being the rightmost.

- [x] For Mobile view only:
    - [x] Reduce the size of the H1 “Record new case contact” to size H2
    - [x] Reduce the size of the H2 “Contact Details” to size H3

### How will this affect user permissions?
- Volunteer permissions: n/a
- Supervisor permissions: n/a
- Admin permissions: n/a

### How is this tested? (please write tests!) 💖💪

Since this is a style update, we checked in our local development environment that the changes were correctly implemented. Refer to screenshots below

### Screenshots please :)

**Desktop view showing the new outline colour for checkboxes and radio buttons, as well as the aligned buttons**
screenshot

**Mobile view showing the reduced heading size**
mobile-screenshot

**Mobile view showing the button layout is per the Figma file provided (different from the desktop view)**
mobile-screenshot-button-layout

### CODE REVIEW

**Comment by compwron** [comment]
@Natblow please resolve merge conflicts so we can merge :)

**Comment by compwron** [comment]
I'm sorry that I let this sit so long! :(

**Comment by compwron** [comment]
I tried to resolve the merge conflicts - let's see how CI goes and fix-forwards if needed.

**Comment by jp524** [comment]
I merged `main` again and the CI passes now! Should be good to go @compwron.

File: app/components/form/title_component.html.erb [hunk 1]
```diff
@@ -1,14 +1,16 @@
 <div class="title-wrapper pt-30">
   <div class="title mb-30">
     <div class="align-items-baseline d-md-flex gap-3">
-      <h1><%= @title %></h1>
+      <h1 class="case-contacts-form desktop-title"><%= @title %></h1>
+      <h2 class="case-contacts-form mobile-title"><%= @title %></h2>
```
Comment (elasticspoon) [inline app/components/form/title_component.html.erb:5]: I know the issue says to change "H1 to size H2", but we shouldn't change the semantic HTML to achieve that. Keep it is an H1 and just change the size of the text in the @media rule.

File: app/components/form/title_component.html.erb [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
       <% if @autosave %>
         <small data-autosave-target="alert"></small>
       <% end %>
     </div>
 
     <div class="row align-items-center mt-5">
-      <h2 class="col-12 col-md-6"><%= @subtitle %></h2>
+      <h2 class="col-12 col-md-6 case-contacts-form desktop-title"><%= @subtitle %></h2>
+      <h3 class="col-12 col-md-6 case-contacts-form mobile-title"><%= @subtitle %></h3>
```
Comment (elasticspoon) [inline app/components/form/title_component.html.erb:13]: same with these h tags

**Review by elasticspoon — CHANGES_REQUESTED** [review]
Hey! Sorry for taking so long to review, really appreciate the work. :heart: 

What you did was great except for a minor fix. Semantically `h1` and `h2` tags mean different things but in our situation we just want to sizes of the items to be different between web and mobile. So instead of changing the size by changing the tag we should keep the tag the same and just style it differently.

**Review by elasticspoon — APPROVED** [review]
Thank you!

**Pull request merged**